articles/storage/storage-python-how-to-use-table-storage.md

```diff
@@ -181,7 +181,7 @@ table_service.delete_table('tasktable')
 Now that you've learned the basics of Table storage, follow these links
 to learn more.
 
-* [Python Developer Center](/develop/python/)
+* [Python Developer Center](https://azure.microsoft.com/en-us/develop/python/)
```

Comment:
@asears Hi Andrew, checking in on this PR. If you wish to proceed, please make the requested change and I will review and accept if all looks good. The other option is to close this PR, and instead make the change in #464 which contains the same modification. Thanks! --Marsh